### PR TITLE
flake8 fixes

### PR DESCRIPTION
**Pull request by Aniket21mathur**

### CODE REVIEW

**Comment by afeena** [comment]
Hi @Aniket21mathur sorry for the delay

I've checked your pr briefly and added some comments :)

File: snare/tanner_handler.py [hunk 1]
```diff
@@ -68,7 +69,8 @@ def create_data(self, request, response_status):
         content = None
         status_code = 200
         headers = {}
-        # Creating a regex object for the pattern of multiple contiguous forward slashes
+        # Creating a regex object for the pattern of multiple contiguous
+        # forward slashes
```
[the remark this answers is not in the document]
Comment (Aniket21mathur) [reply snare/tanner_handler.py]: I was strictly sticking to the flake8 limits. Though I think a comment should be in a single line, I will revert it. Thanks!

**Comment by Aniket21mathur** [comment]
@afeena thanks for reviewing. I made some changes as you suggested.

**Comment by afeena** [comment]
@Aniket21mathur could you please resolve conflicts?

**Comment by Aniket21mathur** [comment]
@afeena sure I will do that

**Comment by Aniket21mathur** [comment]
@afeena rebased with the master. Sorry for the delay. Thanks!

File: bin/snare [hunk 2]
```diff
@@ -92,7 +95,12 @@ def compare_version_info(timeout):
             print_color('timeout fetching the repository version', 'ERROR')
         else:
             if diff_list:
+<<<<<<< HEAD
```
Comment (afeena) [inline bin/snare]: I guess it should not be here :)

File: bin/snare [hunk 3]
```diff
@@ -176,26 +234,38 @@ if __name__ == '__main__':
 
     with open(os.path.join(full_page_path, 'meta.json')) as meta:
         meta_info = json.load(meta)
+<<<<<<< HEAD
```
Comment (afeena) [inline bin/snare]: And this

Comment (Aniket21mathur) [reply bin/snare]: @afeena thanks for pointing out made the changes.

**Pull request merged**